src/EmployerSchema.js

```diff
@@ -0,0 +1,26 @@
+module.exports = (conn) => {
+  const EmployerSchema = new conn.Schema({
```

Comment:
Since documents with this schema represent the relationship between `user` and `organization` would it make sense to call it something more generic like `OrganizationMembershipSchema`... I would believe that a user could a member of an organization other than a "hiring" company... for instance, right now we do not have such relationships as we only have one org in the database (Laboratoria), but adding this collection would imply also adding `OrganizationMembership`s for Laboratoria staff? I think the data involved is generic enough (position and phone number within the context of the organization)... :thinking: ...

Reply:
nice point @lupomontero , I liked `OrganizationMemberships`, more generic